Makefile

```diff
@@ -120,12 +164,3 @@ validate-go-version:
 		exit 1; \
 	fi
 
-# local-env can be used to interact with the local development environment
-# eg:
-#    make local-env-up          # Bring up a basic test environment
-#    make local-env-down        # Tear down the basic test environment
-#    make local-env-nginx-up    # Bring up an nginx based test environment
-#    make local-env-nginx-down  # Tead down the nginx based test environment
-.PHONY: local-env-%
-local-env-%:
```

Comment:
Why is this dropped?

Reply:
@JoelSpeed I never used it and don't like the idea of triggering another Makefile from within this Makefile. 

For local tests I usually have to touch and modify something in the contrib/local-environment directory anyways and just use the Makefile in that directory directly.

Reply:
I used to use this a lot personally, but, I'm ok removing it if no one else does, guess I can just run the `make -C contrib/local-environment` bit myself.

Is there any documentation that refers to folks using this?

Reply:
The only documentation about the contrib directory is on the Contribution page:
https://oauth2-proxy.github.io/oauth2-proxy/community/contribution

But it's not even mentioning the Makefile...

Reply:
Updated the docs :)